### PR TITLE
Swift 5.0

### PR DESCRIPTION
**Pull request by norio-nomura**
This also resolves #141.
Swift 4.2 or later is required.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ aliases:
   - &cocoapods
     script:
       - rm .swift-version
+      - gem install cocoapods
```
Comment (ikesyo) [inline .travis.yml]: I think CocoaPods is pre-installed on Travis (on `language: objective-c`).

Comment (ikesyo) [reply .travis.yml]: Do you mean upgrading CocoaPods? If so `gem update cocoapods` should be appropriate.

Comment (norio-nomura) [reply .travis.yml]: Yup, it looks [works](https://travis-ci.org/Carthage/Commandant/jobs/502880236#L663-L674) as I expected.

File: Commandant.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -414,9 +415,9 @@
 				CLANG_WARN_SUSPICIOUS_MOVE = YES;
 				CURRENT_PROJECT_VERSION = 1;
 				ENABLE_TESTABILITY = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```
Comment (mdiep) [inline Commandant.xcodeproj/project.pbxproj]: Is this change necessary? I know that Nimble requires 10.10, but we should be able to adjust the deployment target for just the test target and leave the framework so it can be deployed to 10.9.

**Comment by norio-nomura** [comment]
Rebased to #141 and force pushed.

**Comment by mdiep** [comment]
Thanks for taking care of this!
